code/modules/jobs/job_types/lawyer.dm

```diff
@@ -42,5 +42,8 @@
 	var/datum/job/lawyer/J = SSjob.GetJobType(jobtype)
 	J.lawyers++
 	if(J.lawyers>1)
-		uniform = /obj/item/clothing/under/rank/civilian/lawyer/purpsuit
+		if(H.jumpsuit_style == PREF_SKIRT)
+			uniform = /obj/item/clothing/under/rank/civilian/lawyer/purpsuit/skirt
+		else
+			uniform = /obj/item/clothing/under/rank/civilian/lawyer/purpsuit
```

Comment:
This is somewhat handled already by https://github.com/tgstation/tgstation/blob/81d585634f7b6b7c8533a138f94b53611db37edd/code/modules/jobs/job_types/_job.dm#L221-L226

I think the better way to go about this is to dynamically set `uniform` to `/obj/item/clothing/under/rank/civilian/lawyer/purpsuit` at the end of `pre_equip`, meaning it retains the behavior of the first lawyer having a unique suit.

Reply:
I pushed a commit per our discussion.